src/send/saga.ts

```diff
@@ -174,7 +174,7 @@ function* sendPaymentLegacy(
     switch (currency) {
       case Currency.Celo: {
         yield put(
-          transferGoldToken({
+          transferGoldTokenLegacy({
```

Comment:
making it clear that these are legacy functions waiting on other cleanup to be removed